app/src/main/java/com/example/atheneum/activities/BookInfoActivity.java

```diff
@@ -198,13 +290,33 @@ public void onChanged(@Nullable Photo photo) {
             @Override
             public void onChanged(final @Nullable Book book) {
                 if (book != null) {
+
+                    bOok = book;
```

Comment:
Okay please rename "b0ok" to something else this is just confusing. Or rename the parameter for onChanged.